server/src/internalClusterTest/java/org/opensearch/cluster/SimpleClusterStateIT.java

```diff
@@ -166,11 +166,10 @@ public void testIndexTemplates() throws Exception {
             .preparePutTemplate("foo_template")
             .setPatterns(Collections.singletonList("te*"))
             .setOrder(0)
-            .addMapping(
-                "type1",
+            .setMapping(
                 XContentFactory.jsonBuilder()
                     .startObject()
-                    .startObject("type1")
+                    .startObject("_doc")
```

Comment:
I think the `_doc` part is not needed

Reply:
Thank you @reta for your comment. Yes, it looks like _doc is not needed.

I will remove all occurrences of _doc added as part of this commit.

Reply:
:+1: